docs/prover/cli/options.md

```diff
@@ -726,19 +726,26 @@ reports may not be generated.
 (-solver)=
 #### `--settings -solver=<solver spec>`
 
-This option sets the SMT solvers being used within the Prover.  By default, a
-portfolio of various different solvers is used.  It can be useful to specify
-only a subset of these to save on computation time.  In rare cases, solver
-specific options can improve performance as well.
-
-The `solver spec` can be a single solver (`-solver=z3`) or a list of solvers
-(`-solver=[cvc5,z3]`), where each such solver can be further modified.  For
-example, `cvc5` refers to the default configuration of `cvc5` whereas
-`cvc5:nonlin` is better for nonlinear problems.  Additional options can be set
-via `z3{randomSeed=17}`.
+By default, a portfolio of SMT solvers using various configurations is used within
+the Prover.  It can be useful to specify only a subset of these to save on
+computation time.  In rare cases, solver specific options can improve performance
+as well.  Setting `-solver=<solver spec>` filters the predefined portfolio to only
+use those configuration that match the given solver specification.
+
+The `solver spec` can be a single solver (`-solver=z3:def`), or a list of solver
```

Comment:
can we list all the portfolios?

Reply:
I don't think this makes a lot of sense, because it will be outdated anyway. I'd rather make sure that we print the portfolio that is being used in the logs so that a user can figure it out on whatever CVT version is being used.
Also, it is a bit more subtle than described here: we actually have a matrix of portfolios right now, based on the logic (LIA, NIA, BV) and the timeout (<5s, <10s, >10s).

Reply:
I've had a quick look how we could allow a user to inspect the solver configs... it is not clear to me how to do this... still I think we should not put a (soon outdated) list here.